docs/rum_collection/crash_reporting.md

```diff
@@ -73,32 +78,70 @@ Datadog.initialize(
 Global.rum = RUMMonitor.initialize()
 ```
 
-### Symbolicate reports
+## Symbolicate crash reports
 
-If your iOS error is unsymbolicated, upload your dSYM file using one of the following tools to symbolicate your different stack traces. For any given error, you have access to the file path, the line number, and a code snippet for each frame of the related stack trace.
+Crash reports are collected in a raw format and mostly contain memory addresses. To map these addresses into legible symbol information, Datadog requires dYSM files, which are generated in your application's build or distribution process.
 
-#### Fastlane Plugin
+### Find your dYSM file
+
+Every iOS application produces dYSM files for each application module. Each application version contains a set of dYSM files.
 
-The Datadog plugin helps you upload dSYM files to Datadog from your fastlane configuration.
+Depending on your setup, you may need to download dSYM files from App Store Connect or find them on your local machine. 
+
+| Bitcode Enabled | Description                                                                                                                                                                                                                                                                                       |
+|-----------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| Yes             | dSYM files are available once [App Store Connect][6] completes processing your application's build.                                                                                                                                                                                                    |
+| No              | Xcode exports dSYM files to `$DWARF_DSYM_FOLDER_PATH` at the end of your application's build. Ensure that the `DEBUG_INFORMATION_FORMAT` build setting is set to **DWARF with dYSM File**. By default, Xcode projects only set `DEBUG_INFORMATION_FORMAT` to **DWARF with dSYM File** in release. |
+
+### Upload your dYSM file
+
+dYSM files minimize an application's binary size and enable faster download speed. By uploading your dYSM file to Datadog, you gain access to the file path, line number, and code snippet of each frame in an error's related stack trace.
```

Comment:
```suggestion
By uploading your dYSM file to Datadog, you gain access to the file path, line number, and code snippet of each frame in an error's related stack trace.
```

nit: It feels out of the context in this instruction. I think this sentence fits perfectly what we say above in "Find your dYSM file", WDYT?
```diff
### Find your dYSM file

Every iOS application produces dYSM files for each application module. 
+ These files minimize an application's binary size and enable faster download speed.
Each application version contains a set of dYSM files.
```